src/commands/kol/whois.ts

```diff
@@ -21,13 +21,27 @@ export const data = new SlashCommandBuilder()
       .setName("player")
       .setDescription("The name or id of the KoL player you're looking up.")
       .setRequired(true)
+      .setMaxLength(30)
   );
 
 export async function execute(interaction: ChatInputCommandInteraction) {
   const playerNameOrId = interaction.options.getString("player", true);
 
   await interaction.deferReply();
 
+  if (
+    // Player names must be betwixt 3 and 30 characters; player IDs can be short, but not long
+    (3 > playerNameOrId.length && playerNameOrId.match(/[^\d]/)) ||
+    // Player names cannot start with numbers
+    playerNameOrId.match(/^\d[^\d]*/) ||
+    // Player names must only contain alphanumeric characters, spaces, and underscores
+    playerNameOrId.match(/[^a-zA-Z0-9_ ]/)
+  ) {
+    await interaction.editReply(
+      "That isn't a player. I can't believe you would try to trick me, your friendly neighborhood OAF, that this string represents a player. I'm very disappointed in you."
```

Comment:
after further consideration i think this is slightly too long and circuitous. how about:

"Come now, you know that isn't a player. Can't believe you'd try and trick me like this. After all we've been through? :pensive: "

we could also do an /8ball thing and make a list of "OAF Disappointment Messages" that we randomly draw from when people try to trick oaf. there were a few instances of people (super and nas i think) trying to get oaf to print a bunch of arbitrary bullshit with /wiki too. still, this is fine for a first pass and we can update this later.